### ISSUE TITLE
Support for Docker EE

### ISSUE THREAD

**Issue by ChetanTekur** [NVIDIA/nvidia-docker#846]
_The template below is mostly useful for bug reports and support questions. Feel free to remove anything which doesn't apply to you and add more information where it makes sense._

_Also, before reporting a new issue, please make sure that:_

- _You read carefully the [documentation and frequently asked questions](https://github.com/NVIDIA/nvidia-docker/wiki)._
- _You [searched](https://github.com/NVIDIA/nvidia-docker/issues?utf8=%E2%9C%93&q=is%3Aissue) for a similar issue and this is not a duplicate of an existing one._
- _This issue is not related to [NGC](https://github.com/NVIDIA/nvidia-docker/wiki/NGC), otherwise, please use the [devtalk forums](https://devtalk.nvidia.com/default/board/200/nvidia-gpu-cloud-ngc-users/) instead._
- _You went through the [troubleshooting](https://github.com/NVIDIA/nvidia-docker/wiki/Troubleshooting) steps._

---

### 1. Issue or feature description
Requesting support for Docker ee version number 17.06.2-ee-16 and/or 18.03.1-ee-4 for RHEL 7.5 (Maipo)
### 2. Steps to reproduce the issue
yum install -y nvidia-docker2

Currently giving the error: 
Error: Package: nvidia-docker2-2.0.3-1.docker18.06.1.ce.noarch (nvidia-docker)
           Requires: docker-ce = 18.06.1.ce

### 3. Information to [attach](https://help.github.com/articles/file-attachments-on-issues-and-pull-requests/) (optional if deemed irrelevant)

 - [ ] Kernel version from `uname -a`
RHEL 7.5 (Maipo)
 - [ ] Any relevant kernel output lines from `dmesg`
Error: Package: nvidia-docker2-2.0.3-1.docker18.06.1.ce.noarch (nvidia-docker)
           Requires: docker-ce = 18.06.1.ce
 - [ ] Driver information from `nvidia-smi -a`
 - [ ] Docker version from `docker version`
Docker ee version number 17.06.2-ee-16 and/or 18.03.1-ee-4
 - [ ] NVIDIA packages version from `dpkg -l '*nvidia*'` _or_ `rpm -qa '*nvidia*'`
 - [ ] NVIDIA container library version from `nvidia-container-cli -V`
 - [ ] NVIDIA container library logs (see [troubleshooting](https://github.com/NVIDIA/nvidia-docker/wiki/Troubleshooting))
 - [ ] Docker command, image and tag used

**Comment by bneil** [comment]
im also running into this issue using docker-ee 17.06.2-ee-16 under centos

**Comment by RenaudWasTaken** [comment]
Hello!

You can still use docker-ee and install the package `nvidia-container-runtime` (not `nvidia-docker2), then you register the runtime manually:
https://github.com/NVIDIA/nvidia-container-runtime#docker-engine-setup

The following command line *should* install the package:
```
> DOCKERVER=$(docker version --format '{{.Server.Version}}' | sed -r 's/-.*//')
> sudo yum install -y "nvidia-container-runtime-2.0.0-1.docker$DOCKERVER.x86_64"
```

**Comment by RenaudWasTaken** [comment]
Note that the support story here is "best effort".

It doesn't really make sense for us to support docker-ee as using a runtime that isn't the default one means you won't get support from docker.
